src/navigation/ProfileStackNavigator.js

```diff
@@ -5,11 +5,16 @@ import { StackNavigator } from 'react-navigation';
 import { colors, textStyles } from '../helpers/styles';
 
 import ProfileScreen from '../screens/ProfileScreen';
+//import SignInScreen from '../screens/SignInScreen';
```

Comment:
Why are you committing the codes you have already commented. Most probably you have forgotten to remove these. As I see you have added SignInScreen to SignStackNavigator.